backend-app/controllers/users_controllers/admin_controller.ts

```diff
@@ -178,17 +113,22 @@ export const banUser = async (req: IReq, res: IRes, next: INext) => {
             throw new AppError(400, 'You cannot ban an admin');
         user.accessRestricted = true;
         await user.save();
-        res.status(200).json({
+        this.setStatus(200);
+        return {
             message: 'User is now banned',
-        });
-    } catch (err) {
-        next(err);
+        };
     }
-};
-
-export const unbanUser = async (req: IReq, res: IRes, next: INext) => {
-    const { userId } = req.params;
-    try {
+    @Example({
+        message: 'User is now unbanned',
+    })
+    @Response(400, 'User is not banned')
+    @Response(400, 'You cannot unban yourself')
```

Comment:
this will not work, I have stied it but it only shows the last one, the way i did it is like this :
```js
    @Response(
        400,
        `- This route is not for role updates. Please use /updateRole\n\- This route is not for password updates. Please use auth/updateMyPassword`
    )
    
```
it is long, I know but this is best i could do, try to update it and if you found a better solution tell me

Reply:
okay